client/my-sites/drafts/README.md

```diff
@@ -0,0 +1,4 @@
+Drafts
+======
+
+The view module for the Drafts post list in `/posts/drafts` as a React component. See also the single `Draft` component.
```

Comment:
This actually powers the drafts panel in the editor and `/drafts`, not `/posts/drafts`. The latter is via `client/my-sites/posts`.

So this folder has both the section routes for `/draft`, the controller, the main view for the sections, and a list component that we reuse in the editor. It's a bit messy. :)